apps/server/src/modules/files-storage/controller/api-test/files-storage-download-upload.api.spec.ts

```diff
@@ -376,35 +377,64 @@ describe('files-storage controller (API)', () => {
 		});
 
 		describe(`with valid request data`, () => {
-			const setup = async () => {
-				const { result: uploadedFile } = await api.postUploadFile(`/file/upload/${validId}/schools/${validId}`);
-				const expectedResponse = TestHelper.createFile('bytes 0-3/4');
+			describe('when mimetype is not application/pdf', () => {
+				const setup = async () => {
+					const { result: uploadedFile } = await api.postUploadFile(`/file/upload/${validId}/schools/${validId}`);
+					const expectedResponse = TestHelper.createFile({ contentRange: 'bytes 0-3/4' });
```

Comment:
I think it would be better to set the mimetype here explicitly and not rely on the default value in the test-helper. It would make the test more explicit and more robust.